lib/plugins/encrypted_parameters/kms_key.rb

```diff
@@ -13,10 +15,24 @@ def initialize(arn)
         def self.create
           resp = Aws::KMS::Client.new.create_key
           arn = resp.key_metadata.arn
-
           new(arn)
         end
 
+        def update
+          standard_tags=get_tags
+          resp = @kms_client.tag_resource({
+            key_id: @arn, # arn of the CMK being tagged
+            tags: standard_tags, # A list of tags.
+          })
+        end
+
+        def get_tags
+          tags=Moonshot::Stack.make_tags(Moonshot.config)
+          formated_tags=[]
+          tags.each{ |tag| formated_tags << { tag_key: tag[:key], tag_value: tag[:value]}}
```

Comment:
These three lines can be rewritten as tags.map { |tag| { tag_key: tag[:key], tag_value: tag[:value]} }